### PR TITLE
bug #765: Fixed function parsing issue in case function's complete body is in one single line

### PR DESCRIPTION
**Pull request by sanyamsinghal**

### CODE REVIEW

File: migtests/lib/yb.py [hunk 1]
```diff
@@ -176,7 +176,17 @@ def fetch_all_function_names(self, schema_name="public") -> List[str]:
 		cur.execute(f"SELECT routine_name FROM information_schema.routines WHERE routine_schema = '{schema_name}'")
 		return [function[0] for function in cur.fetchall()]
 
-	def fetch_all_table_rows(self, table_name, schema_name="public") -> set[str]:
+	def fetch_all_table_rows(self, table_name, schema_name="public") -> any:
```
Comment (amit-yb) [inline migtests/lib/yb.py:179]: Specifying `any` is as good as specifying nothing about the return value.
What made you replace the `set[str]` to `any`?

File: yb-voyager/cmd/analyzeSchema.go [hunk 2]
```diff
@@ -676,12 +676,12 @@ func createSqlStrInfoArray(path string, objType string) []sqlInfo {
 
 		// Assuming that both the dollar quote strings will not be in same line
 		if dollarQuoteFlag == 0 {
-			if matches := dollarQuoteRegex.FindStringSubmatch(currLine); matches != nil {
-				dollarQuoteFlag = 1 //denotes start of the code/body part
-				codeBlockDelimiter = matches[0]
-			} else if strings.Contains(currLine, ";") { // in case, there is no body part
+			if strings.Contains(currLine, ";") { // in case, there is no body part or body part is in single line
```
Comment (amit-yb) [inline yb-voyager/cmd/analyzeSchema.go:679]: What if there is a string literal containing `;` in it? Will it trigger false detection of end of function? How about checking if last non-white-space character is semi-colon?

Comment (sanyamsinghal) [reply yb-voyager/cmd/analyzeSchema.go:679]: yeah, nice catch. Will make that change.

Comment (sanyamsinghal) [reply yb-voyager/cmd/analyzeSchema.go:679]: but what if there is a sql comment after the semi-colon

**Review by amit-yb — APPROVED** [review]
LGTM.

**Pull request merged**